sematic/db/models/run.py

```diff
@@ -157,7 +157,7 @@ def convert_tags_to_json(self, _, value) -> str:
     @validates("description")
     def strip_description(self, _, value) -> str:
         if value is not None:
-            value = re.sub(r"\n\s{4}", "\n", value.strip())
+            value = re.sub(r"\n( {4}|\t)", "\n", value.strip())
```

Comment:
Instead of using `\s`, I use either explicit 4 whitespaces or a single tab. This way we exclude `\n` from the charset of `\s`

Reply:
Out of curiosity, I also tried something else, for simplicity I didn't use that version.
 
```python
def strip_description(self, _, value) -> str:
        if value is not None:
            # value = re.sub(r"\n( {4}|\t)", "\n", value.strip())
            
            lines = value.splitlines()
            # convert tabs into spaces
            lines = [re.sub(r"^[ \t]+\b", 
                lambda m: m.group(0).replace("\t", "    "), line) 
                for line in lines]

            # check the length of indentation of each line
            indent_lens = [(m := re.match(r"^ +\b", line)) and len(m.group(0)) or 0 
                for line in lines if len(line.strip()) != 0]

            # find the shortest indentation
            cut_off = min(indent_lens)

            # vertical clip at the `cut-off` limit for each line
            lines = [line[min(cut_off, len(line)):] for line in lines]

            value = "\n".join(lines)

        return value
```

This algorithm scans each line to get the minimum indentation, then makes a vertical cut-off at the shortest indentation. It helps with the case that the indentation is not always 4 spaces, e.g. from a DocString of a nested function.

If this is the preferred solution, please let me know.